### PR TITLE
Add RoyalTS 6.00.50614.0

### PR DESCRIPTION
**Pull request by al-lac**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17929)

### CODE REVIEW

**Comment by al-lac** [comment]
@KevinLaMS Somehow there is no reviewer assigned for this PR. Can you check please?

File: manifests/r/RoyalApps/RoyalTS/6.00.50614.0/RoyalApps.RoyalTS.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.00.50614.0
```
Comment (OfficialEsco) [inline manifests/r/RoyalApps/RoyalTS/6.00.50614.0/RoyalApps.RoyalTS.yaml:3]:
```suggestion
PackageVersion: 6.0.50614.0
```

File: manifests/r/RoyalApps/RoyalTS/6.00.50614.0/RoyalApps.RoyalTS.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+PackageName: RoyalTS
```
Comment (OfficialEsco) [inline manifests/r/RoyalApps/RoyalTS/6.00.50614.0/RoyalApps.RoyalTS.yaml:4]:
```suggestion
PackageName: Royal TS
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Microsoft don't do reviews anymore, they have given 3 of us top contributors that job to ensure quality...
Anyways, the metadata needs to match whatever Windows Control Panel sees
image

**Comment by al-lac** [comment]
@OfficialEsco thanks a lot for the update about reviews and suggestions for the metadata. 👍

**Comment by OfficialEsco** [comment]
> @OfficialEsco thanks a lot for the update about reviews and suggestions for the metadata. 👍

Uhoh you didn't change the foldername 😬

**Comment by al-lac** [comment]
Ahh. Will do that right away.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**